demystifying_geospatial_data/demystifying_geospatial_data.md

```diff
@@ -0,0 +1,221 @@
+<!--
+
+author:   Elizabeth Drellich
+email:    drelliche@chop.edu
+version:  1.0.0
+module_template_version: 2.0.1
+language: en
+narrator: UK English Female
+title: Module Title
+comment:  This module is a brief introduction to geospatial (location) data.
+long_description: This module will survey some of the benefits of using geospatial data for research purposes. No previous exposure to geospatial data is expected. If you have any interest in maps or are wondering if using geospatial data might be helpful for your work, this lesson is designed to help you decide whether learning more about geospatial techniques is right for you and your project.
+estimated_time:  20 minutes
+
+@learning_objectives  
+
+After completion of this module, learners will be able to:
+
+- Identify geospatial data
+- Describe some of the benefits of using geospatial data
+- Recognize some of the issues you may encounter when using geospatial data
+
+@end
+
+link:  https://chop-dbhi-arcus-education-website-assets.s3.amazonaws.com/css/styles.css
+
+script: https://kit.fontawesome.com/83b2343bd4.js
+
+-->
+
+# Demystifying Geospatial Data
+
+<div class = "overview">
+
+## Overview
+@comment
+
+**Is this module right for me?** @long_description
+
+**Estimated time to completion:** @estimated_time
+
+**Pre-requisites**: none
+
+**Learning Objectives**
+
+@learning_objectives
+
+</div>
+
+## What is Geospatial Data?
+
+If you have ever look at a map, followed GPS directions, or even just used a cell phone to check the time, you have used geospatial data.
+
+**Geospatial Data** is any data that includes information about the location of an object or event on the Earth. We think of these locations as being on the Earth's surface, but they might actually be a bit above the surface, like weather or airplane flight paths, or below, like oil deposits.
```

Comment:
I was holding this for the next module, but I agree it should go here!

Reply:
added!

Reply:
@pm0kjp would it make sense to have a small map here (thinking Roberts building as a point, South St as a line segment, and 19146 as a polygon)? I want to make such an image for the other module I'm working on focusing on these types of data, so it might make sense to put a visual here?